### PR TITLE
python3Packages.einops: init at 0.3.2

### PR DESCRIPTION
**Pull request by yl3dy**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by yl3dy** [comment]
Result of `nixpkgs-review pr 137517` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python38Packages.einops</li>
    <li>python39Packages.einops</li>
  </ul>
</details>

File: pkgs/development/python-modules/einops/default.nix [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    mxnet
+    tensorflow
+  ];
+
+  # No CUDA in sandbox
+  EINOPS_SKIP_CUPY = 1;
+
+  checkPhase = ''
+    export HOME=$TMPDIR
+    python -m nose tests -vds
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/einops/default.nix:45]:
```suggestion
    ${python.interpreter} -m nose tests -vds
```
or can we use nosetests?

Comment (yl3dy) [reply pkgs/development/python-modules/einops/default.nix]: Totally forgot about nosetests. Yes, it absolutely can be used, fixed the expression.

**Comment by SuperSandro2000** [comment]
@ofborg build python3Packages.einops

**Comment by risicle** [comment]
Tests appear to hang for me, nixos x86_64

**Review by samuela — APPROVED** [review]

**Pull request merged**